src/openqaoa-core/algorithms/rqaoa/rqaoa_workflow.py

```diff
@@ -51,17 +51,17 @@ class RQAOA(Workflow):
         For a complete list of its parameters and usage please see the method set_circuit_properties
     rqaoa_parameters: `RqaoaParameters`
         Set of parameters containing all the relevant information for the recursive procedure of RQAOA.
-    results: `RQAOAResults`
+    results: `RQAOAResul`
```

Comment:
Deleted the `t` in all RQAOAResult in this file :x

Reply:
Fixed